states/RI/governor.yaml

```diff
@@ -0,0 +1,60 @@
+contact_form:
+  steps:
+    - visit: "http://www.governor.ri.gov/contact/"
+    - fill_in:
+        - name: "firstname"
+          selector: "#firstname"
+          value: $NAME_FIRST
+          required: true
+        - name: "lastname"
+          selector: "#lastname"
+          value: $NAME_LAST
+          required: true
+        - name: "email"
+          selector: "#email"
+          value: $EMAIL
+          required: true
+        - name: "area"
+          selector: "#area"
+          value: $PHONE_AREA_CODE
+          required: true
+          # max length: 3
+        - name: "exchange"
+          selector: "#exchange"
+          value: $PHONE3
+          required: true
+          # max length: 3
+        - name: "phnum"
+          selector: "#phnum"
+          value: $PHONE4
+          required: true
+          # max length: 4
+        - name: "address1"
+          selector: "#address1"
+          value: $ADDRESS_STREET
```

Comment:
You can put `required: false` if you want. I don't really think the schema specifies, but that would be more explicit.

Reply:
I think about that when I see fields that we should fill out but aren't required by the form.  I'll adapt to explicitly state false where we want to include data not required by the form.  Thanks.